pymatgen/analysis/diffraction/tem.py

```diff
@@ -88,10 +95,10 @@ class TEMCalculator(AbstractDiffractionPatternCalculator):
     some important constants needed for calculations
     """
     planck = 6.62607004 * 10**-34
-    restMassElec = 9.1093835611 * 10**-31
-    chargeElec = 1.60217662 * 10**-19 
-    speedLight = 299792458
-    vacuumPerm = 8.8541878176 * 10**-12
+    rest_mass_elec = 9.1093835611 * 10**-31
+    charge_elec = 1.60217662 * 10**-19 
+    speed_light = 299792458
+    vacuum_perm = 8.8541878176 * 10**-12
```

Comment:
These should not be hard coded. Since scipy is a dependency, you can use scipy.constants for all of these. Their values are far more accurate and updated.